src/server/hotkey_collector.h

```diff
@@ -102,6 +102,7 @@ class internal_collector_base : public dsn::replication::replica_base
     explicit internal_collector_base(replica_base *base) : replica_base(base){};
     virtual void capture_data(const dsn::blob &hash_key, uint64_t weight) = 0;
     virtual void analyse_data(detect_hotkey_result &result) = 0;
+    virtual void clear() = 0;
```

Comment:
Where you call this function?

Reply:
https://github.com/apache/incubator-pegasus/pull/632/files#diff-7d14c5784befda2ae449c643a1ec2dd2b9988db5fc72325f48e855b6a566e02eR126
In another PR